### PR TITLE
Update unit tests

### PR DESCRIPTION
**Pull request by dughetti**

### CODE REVIEW

**Comment by lsvx** [comment]
Nicolas, this looks awesome! We still need to clean the DB of test nodes and edges after the testing is done so as to not pollute the graph. Also, do we use nock anymore? If not, let's eliminate it entirely.

File: tests/root_hooks.js [hunk 1]
```diff
@@ -57,11 +45,25 @@ beforeEach(function(done) {
 });
 
 after(function (done) {
```
Comment (dughetti) [inline tests/root_hooks.js]: @lsvx here I delete the test nodes. :+1:

Comment (lsvx) [inline tests/root_hooks.js]: Ah very cool! What do you think about cleaning Redis?

Comment (dughetti) [inline tests/root_hooks.js]: It is not necessary. Before each test we are disabling cache.

Comment (lsvx) [inline tests/root_hooks.js]: :D awesome

**Comment by dughetti** [comment]
@lsvx , @maxicecilia I proceed with the merge..

**Comment by maxicecilia** [comment]
+1 looks great

**Pull request merged**